shared/src/commonTest/kotlin/co/nimblehq/kmm/template/data/repositories/RepositoryTest.kt

```diff
@@ -16,33 +16,33 @@ import kotlin.test.Test
 class RepositoryTest {
 
     @Mock
-    private val mockUserRemoteDataSource = mock(UserRemoteDataSource::class)
+    private val mockRemoteDataSource = mock(RemoteDataSource::class)
 
     private lateinit var repository: Repository
 
     @BeforeTest
     fun setup() {
-        repository = RepositoryImpl(mockUserRemoteDataSource)
+        repository = RepositoryImpl(mockRemoteDataSource)
     }
 
     @Test
     fun `when calling getUsers successfully - it returns users`() = runTest {
```

Comment:
We did not make changes here.

Reply:
@kaungkhantsoe A repository could contains multiple APIs, not the same to a use case, then we should keep it 🙏